src/Components/Dropdowns/DropdownHrefItem.tsx

```diff
@@ -63,23 +63,24 @@ export const DropdownHrefItem = forwardRef<
     }
 
     return (
-      <button
-        type="button"
-        id={id}
-        ref={ref}
-        style={styles}
-        title={title}
-        className={dropdownItemClass}
-        data-testid={testID}
+      <a
+        href={href}
+        className="cf-dropdown-item--children cf-dropdown-href-text"
       >
-        <DropdownItemSelectionIndicator type={type} />
-        <a
-          href={href}
-          className="cf-dropdown-item--children cf-dropdown-href-text"
+        <button
+          type="button"
+          id={id}
+          ref={ref}
+          style={styles}
+          title={title}
+          className={dropdownItemClass}
+          data-testid={testID}
         >
+          <DropdownItemSelectionIndicator type={type} />
+
           {children}
-        </a>
-      </button>
+        </button>
+      </a>
```

Comment:
make sure you test the results of this in different places than the one you're adding it to, if applicable. We don't want to accidentally break something in a different area of the app

Reply:
Yes that is correct! 

and I tested a few dropdowns, they work fine @hoorayimhelping 👍

Reply:
Good man!